src/client/components/PlaceInfo/PlaceInfo.jsx

```diff
@@ -3,8 +3,9 @@ import { Button, Modal } from 'react-bootstrap';
 import _ from 'lodash';
```

Comment:
I would prefer that we don't keep the lighthouse reports here. I don't have an alternate solution just yet, but I am apprehensive to add these large files that are not code changes in here. Any ideas?